src/Components/CriticalCareRecording/types/CriticalCare_BloodSugar.res

```diff
@@ -0,0 +1,45 @@
+let frequencyOptions = [
+  {
+    "name": "Once a Day(OD)",
+    "value": "OD",
+  },
+  {
+    "name": "Twice a day(BD)",
+    "value": "BD",
+  },
+  {
+    "name": "Thrice in a day(TD)",
+    "value": "TD",
+  },
+]
+
+type t = {
+  blood_sugar_level: string,
+  dosage: string,
+  frequency: string,
+}
+
+let blood_sugar_level = t => t.blood_sugar_level
+let dosage = t => t.dosage
+let frequency = t => t.frequency
+
+let init = {
+  blood_sugar_level: "",
+  dosage: "",
+  frequency: "OD",
+}
+
```

Comment:
we don't need this

Reply:
wouldn't it be cleaner if we have the init records in their respective editor's types file?